### DISCUSSION TITLE
Does GoRelease allow container builds using nerctl build, and buildkit

### DISCUSSION THREAD

**Discussion by karlmutch** [goreleaser/goreleaser#2847] [Q&A] [answered]
Hi,

We are using rancher desktop and would like to use GoReleaser along with the 'nerdctl build' command to build images.

Is this supported in the pro version.

Thanks,
Karl

**Comment by caarlos0** [comment, marked as answer]
not yet, but I'll look into it 

edit: PR -> https://github.com/goreleaser/goreleaser/pull/2884

**Reply by developer-guy** [reply]
@caarlos0 I can take care of that one actually if you want. I have experience on nerdctl, lima, and rancher-desktop a bit.

**Reply by caarlos0** [reply]
Thanks @developer-guy, but no need, I want to learn about all those things :D 

can I ping you if I have questions and/or on the code review? thankss 🙏

**Reply by caarlos0** [reply]
so, pretty much we can simply allow to use `nerdctl` instead of `docker`, as their command lines is compatible, is that correct?

if so, this should work, right? https://github.com/goreleaser/goreleaser/compare/nerdctl?expand=1

**Reply by karlmutch** [reply]
In. theory it probably would.  What I dont know is what the side effects are of using buildkit hosted within a Kubernetes cluster for which a ~/.kube/config file is needed.  I assume thats the responsibility of the developers environment but ....
